test/fast/rollback.test.ts

```diff
@@ -65,24 +69,36 @@ describe("Rollback", function() {
         await setup(param2);
         const gasleft = await rollup.callStatic.testRollback(1);
         console.log("Gas usage out of the loop", gasleft.toNumber());
-        const tx = await rollup.testRollback(1);
+        const batchID = 1;
+        const tx = await rollup.testRollback(batchID);
         const [status] = await rollup.queryFilter(
             rollup.filters.RollbackStatus(null, null, null),
             tx.blockHash
         );
+        const [event] = await rollup.queryFilter(
+            rollup.filters.RollbackTriggered(null, null),
+            tx.blockHash
+        );
```

Comment:
When multiple async (promise) calls are made that don't have data dependencies on each other, it can be a good idea to let them run in parallel especially when the underlying async function is making RPC calls. Note that this should not be done for L1 txns/writes. This can also be done in a few other spots in this test, and can reduce test runtimes.

```suggestion
        const [[status], [event]] = await Promise.all([
            rollup.queryFilter(
                rollup.filters.RollbackStatus(),
                tx.blockHash
            ),
            rollup.queryFilter(
            	rollup.filters.RollbackTriggered(),
            	tx.blockHash
            )
        ]);
```

Reply:
good advice, both of them, will implement 👍🏼